app/controllers/liquid_tags_controller.rb

```diff
@@ -0,0 +1,13 @@
+class LiquidTagsController < ApplicationController
+  before_action :authenticate_user!
```

Comment:
Since this is an internal API and we want to show the result in the editor I decided to only make it accessible to authenticated users.